.github/ISSUE_TEMPLATE/new_bug.md

```diff
@@ -0,0 +1,32 @@
+---
+name: New Bug
+about: Identification of a new issue
+title: 'New Bug'
+labels: 
+assignees: '@abbylindsay'
+
+---
+
+**Describe the bug- why is it wrong?:**
+<!-- A clear and concise description of what you/the participant is seeing and why it is a problem. -->
+
+**How was this bug identified:**
+<!-- Who saw it? What was the context? -->
+
+**How to reproduce?:**
+<!-- If no instructions, you MUST provide an explaination as to why not. -->
+
+**Troubleshooting completed so far:**
+<!-- What has been done thus far to resolve/investigate the bug? -->
+
+**Expected behavior?:**
```

Comment:
Usually this is expected behavior and actual behavior. Maybe should match the columns and terms used in the V&V